fuseops/simple_types.go

```diff
@@ -95,6 +95,12 @@ type InodeAttributes struct {
 	// Ownership information
 	Uid uint32
 	Gid uint32
+
+	// The number of blocks allocated to the file, 512-byte units. (This may be
+	// smaller than Size/512 when the file has holes.)
+	Blocks uint64
+	// The "preferred" blocksize for efficient filesystem I/O.
+	BlockSize uint32
```

Comment:
I know that you're following the convention used by `stat(2)`, but it's quite confusing for this to be called "block size" when the unit of `Blocks` is already defined to be 512 bytes. Please name this `IoSize` for symmetry with [`StatFSOp`](https://godoc.org/github.com/jacobsa/fuse/fuseops#StatFSOp).